OneCommunityGlobal/HighestGoodNetworkApp
# Ivy-submit-btn-permission-assigned

When Assigned permissions and clicked submit the modal is not closing .

<img width="500" alt="Screenshot 2024-07-27 at 5 12 45 PM" src="https://github.com/user-attachments/assets/a936e19f-f80e-4e1b-a4bd-f3d18bf18b76">


## Related PRS (if any):
…

## Main changes explained:
- Delete file A for removing unused components …
- Update file B for including new pattern …
- Create file C for introducing new components …
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. open permission management tab and chose Manage User Permissions, when no permission or any errors, click the submit btn, the modal doesn't close. Submit successfully, the modal will close automatically.

https://github.com/user-attachments/assets/4e0ec4b4-7030-4632-9b89-a0c1988759b8


## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
